### ISSUE TITLE
Exception in thread "main" java.lang.VerifyError: Bad return type

### ISSUE THREAD

**Issue by natros** [querydsl/querydsl#2275]
This is happening to me after upgrading to 4.2.0

```
Exception in thread "main" java.lang.VerifyError: Bad return type
Exception Details:
  Location:
    com/querydsl/sql/dml/AbstractSQLInsertClause.values([Ljava/lang/Object;)Lcom/querydsl/sql/dml/SQLInsertClause; @5: areturn
  Reason:
    Type 'com/querydsl/sql/dml/AbstractSQLInsertClause' (current frame, stack[0]) is not assignable to 'com/querydsl/sql/dml/SQLInsertClause' (from method signature)
  Current Frame:
    bci: @5
    flags: { }
    locals: { 'com/querydsl/sql/dml/AbstractSQLInsertClause', '[Ljava/lang/Object;' }
    stack: { 'com/querydsl/sql/dml/AbstractSQLInsertClause' }
  Bytecode:
    0x0000000: 2a2b b602 74b0                         

	at com.querydsl.sql.AbstractSQLQueryFactory.insert(AbstractSQLQueryFactory.java:73)
	at com.github.natros.Main.main(Main.java:23)
```
Test case
https://github.com/natros/querydsl-test

java version "1.8.0_162"
Java(TM) SE Runtime Environment (build 1.8.0_162-b12)
Java HotSpot(TM) 64-Bit Server VM (build 25.162-b12, mixed mode)
macOS 10.13.3

**Comment by Shredder121** [comment]
@timowest did we miss some bridge annotations?

**Comment by natros** [comment]
It's the same error in https://travis-ci.org/querydsl/querydsl/jobs/355097588 

```
Tests in error: 
  JDBCIntegrationTest.Delete:157->insert:166 Verify Bad return type
Exception De...
  JDBCIntegrationTest.Insert:131->insert:166 Verify Bad return type
Exception De...
  JDBCIntegrationTest.Update:141->insert:166 Verify Bad return type
Exception De...
Tests run: 96, Failures: 0, Errors: 3, Skipped: 0
```

You could ask for testers before releasing

**Comment by timowest** [comment]
Sorry, it appears a broken version made it accidently out, I will prepare a new release

**Comment by Shredder121** [comment]
As we speak Querydsl 4.2.1 has been released, so this is now fixed.
Sorry for the inconvenience! 😅

**Comment by Ebiyejames** [comment]
Hello Shredder,

I am using the java owlapi to add new classes to an existing ontology (owl file) and I am encountering same error. In chat, you said this is fixed. Can you please clarify me how to solve this problem. See error below. Thanking you in advance for your quick response. 

Exception in thread "main" java.lang.VerifyError: Bad return type
Exception Details:
  Location:
    org/semanticweb/owlapi/formats/OWLFunctionalSyntaxOntologyFormatFactory.getNewFormat()Lorg/semanticweb/owlapi/formats/OWLOntologyFormat; @7: areturn
  Reason:
    Type 'org/semanticweb/owlapi/formats/OWLFunctionalSyntaxOntologyFormat' (current frame, stack[0]) is not assignable to 'org/semanticweb/owlapi/formats/OWLOntologyFormat' (from method signature)
  Current Frame:
    bci: @7
    flags: { }
    locals: { 'org/semanticweb/owlapi/formats/OWLFunctionalSyntaxOntologyFormatFactory' }
    stack: { 'org/semanticweb/owlapi/formats/OWLFunctionalSyntaxOntologyFormat' }
  Bytecode:
    0x0000000: bb00 0259 b700 03b0                    

	at org.coode.owlapi.functionalrenderer.OWLFunctionalSyntaxOntologyStorerFactory.getFormatFactory(OWLFunctionalSyntaxOntologyStorerFactory.java:33)
	at org.semanticweb.owlapi.model.OWLOntologyStorerFactoryRegistry.getKey(OWLOntologyStorerFactoryRegistry.java:130)
	at org.semanticweb.owlapi.model.OWLOntologyStorerFactoryRegistry.getKey(OWLOntologyStorerFactoryRegistry.java:63)
	at com.github.ansell.abstractserviceloader.AbstractServiceLoader.add(AbstractServiceLoader.java:103)
	at com.github.ansell.abstractserviceloader.AbstractServiceLoader.<init>(AbstractServiceLoader.java:85)
	at com.github.ansell.abstractserviceloader.AbstractServiceLoader.<init>(AbstractServiceLoader.java:44)
	at org.semanticweb.owlapi.model.OWLOntologyStorerFactoryRegistry.<init>(OWLOntologyStorerFactoryRegistry.java:68)
	at org.semanticweb.owlapi.model.OWLOntologyStorerFactoryRegistry.<clinit>(OWLOntologyStorerFactoryRegistry.java:65)
	at org.semanticweb.owlapi.apibinding.OWLManager.createOWLOntologyManager(OWLManager.java:119)
	at org.semanticweb.owlapi.apibinding.OWLManager.createOWLOntologyManager(OWLManager.java:107)
	at OntologyCreation.main(OntologyCreation.java:20)

**Comment by Shredder121** [comment]
Sorry for the long delay, but it's fixed by using the newest version, not `4.2.0`.
